plugins/modules/aci_contract_subject.py

```diff
@@ -332,6 +365,12 @@ def main():
     state = module.params.get("state")
     tenant = module.params.get("tenant")
     name_alias = module.params.get("name_alias")
+    subject_label = module.params.get("subject_label")
+    subject_label_direction = module.params.get("subject_label_direction")
+
+    subject_label_class = SUBJ_LABEL_MAPPING[subject_label_direction]
```

Comment:
this will error when subject_label_direction is None value, which happens when subject_label_direction is not provided